doc/src/variable.rst

```diff
@@ -996,10 +996,19 @@ via the link in this paragraph.
 
 The label2type(kind,label) function converts type labels into numeric
 types, using label maps created by the :doc:`labelmap <labelmap>` or
-:doc:`read_data <read_data>` commands.  The first argument is the
-label map kind (atom, bond, angle, dihedral, or improper) and the
-second argument is the label.  The function returns the corresponding
-numeric type.
+:doc:`read_data <read_data>` commands.  The first argument is the label
+map kind (atom, bond, angle, dihedral, or improper) and the second
+argument is the label.  The function returns the corresponding numeric
+type or -1 in case the specified label for the given kind of type does
+not exist.
+
+.. versionadded:: TBD
+
+The is_typelabel(kind,label) function has the same arguments as
+label2type(), but returns 1 if the type label has been assigned,
+otherwise it returns 0.  This function can be used to check if a
```

Comment:
@akohlmey I thought the agreement was on -1. Should this be corrected to `otherwise it returns -1`